### PR TITLE
Add healprod

### PR DESCRIPTION
**Pull request by jawadqur**
Copied preprod manifest to prod.

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- portal

**Review by williamhaley — APPROVED** [review]

File: healdata.org/etlMapping.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      - name: object_id
+      - name: md5sum
+      - name: file_name
+      - name: file_size
+      - name: data_format
+      - name: data_type
+      - name: project_id
+    parent_props:
+      - path: core_metadata_collections[cmc_unique_id:submitter_id]
+  - name: preprod-heal_oa-file
```
Comment (haraprasadj) [inline healdata.org/etlMapping.yaml:47]: should this be prod?

File: healdata.org/manifests/hatchery/hatchery.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
+{
+  "user-namespace": "jupyter-pods-healpreprod",
```
Comment (haraprasadj) [inline healdata.org/manifests/hatchery/hatchery.json:2]: should this be prod?

Comment (jawadqur) [reply healdata.org/manifests/hatchery/hatchery.json]: Yeah good catch 👀

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- portal

**Review by haraprasadj — APPROVED** [review]

**Review by ac3eb — APPROVED** [review]

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- portal

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- portal

**Comment by PlanXCyborg** [comment]
# healdata.org/manifest.json
## :warning: Services on branch
- portal

**Pull request merged**